OutOfSchool/OutOfSchool.IdentityServer/ViewModels/ChangeEmailViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.IdentityServer.ViewModels
+{
+    public class ChangeEmailViewModel
+    {
+        [DataType(DataType.EmailAddress)]
+        [Required(ErrorMessage = "Email is required")]
+        public string CurrentEmail { get; set; }
+
+        [DataType(DataType.EmailAddress)]
+        [Required(ErrorMessage = "Email is required")]
+        public string Email { get; set; }
+
+        [DataType(DataType.EmailAddress)]
+        [Required(ErrorMessage = "Email is required")]
+        public string ConfirmEmail { get; set; }
+
+        public string ReturnUrl { get; set; }
```

Comment:
i don't see this field used anywhere. Maybe we can use it in some kind of button in `ConfirmChange.cshtml` to redirect user back to frontend page.